README.md

```diff
@@ -147,13 +148,25 @@ In a default Archivematica installation, this is `/var/archivematica/sharedDirec
 
 An optional parameter `-u` or `--uuid` may be passed to only reindex the AIP that has the matching UUID.
 
-`--delete` will delete any data found in ElasticSearch with a matching UUID before re-indexing.
+`--delete` will delete any data found in Elasticsearch with a matching UUID before re-indexing.
 This is useful if only some AIPs are missing from the index, since AIPs that already exist will not have their information duplicated.
 
-`--delete-all` will delete the entire AIP ElasticSearch index before starting.
+`--delete-all` will delete the entire AIP Elasticsearch index before starting.
 This is useful if there are AIPs indexed that have been deleted.
 This should not be used if there are AIPs stored that are not locally accessible.
 
+### rebuild-transfer-backlog
+
+*Versions*: Archivematica 1.6
+
+rebuild-transfer-backlog will recreate the `transfers` Elasticsearch index from the Transfer Backlog location. This is useful if the Elasticsearch index has been deleted or damaged or during software upgrades where the document schema has changed.
```

Comment:
Minor: The rest of the readme uses [semantic linebreaks](http://rhodesmill.org/brandon/2012/one-sentence-per-line/). Please update this to as well.